app/src/main/java/org/piramalswasthya/sakhi/configuration/InfantRegistrationDataset.kt

```diff
@@ -111,12 +111,13 @@ class InfantRegistrationDataset(
     private var weight = FormElement(
         id = 11,
         inputType = InputType.EDIT_TEXT,
-        title = "Weight at Birth(kg)",
+        title = "Weight at Birth(gram)",
```

Comment:
What happens to old data when we make this change?

Reply:
Yes @drtechie, for that we have to make migrations.
cc: @madhavaramu83

Reply:
@drtechie , I have reversed it back to kg, in DB we are saving it in kg, discussed with @madhavaramu83